examples/regression.py

```diff
@@ -3,18 +3,25 @@
 import functorch
 from functorch._src.compile_utils import strip_overloads
 import iree_torch
-import numpy as np
-from sklearn.datasets import make_regression
-from sklearn.model_selection import train_test_split
 import torch
 import torch_mlir
 
 
 # Dataset
-X, y, coef = make_regression(n_features=3, coef=True)
-X = torch.from_numpy(X).to(dtype=torch.float32)
-y = torch.from_numpy(y).to(dtype=torch.float32)
-X, X_test, y, y_test = train_test_split(X, y)
```

Comment:
train_test_split can be done with a one-liner indexing. Something like
split = int(0.9 * num_examples)
X, X_test = X[:split], X[split:]

Reply:
Yeah, I removed it altogether because I thought it was unnecessary for illustrating the example and ends up just adding visual noise to the code.  I've added it back and done it manually.